sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionProcessorCloseReason.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.EventHubs.Processor
+{
+    /// <summary>
+    ///   The reason for closing an <see cref="IPartitionProcessor" />.
+    /// </summary>
+    ///
+    public enum PartitionProcessorCloseReason
```

Comment:
Should we have an `Unknown` member here, just in case?

Reply:
Why not?